pytorch_lightning/callbacks/lr_monitor.py

```diff
@@ -106,6 +106,7 @@ def on_train_batch_start(self, trainer, *args, **kwargs):
             latest_stat = self._extract_stats(trainer, interval)
 
             if trainer.logger is not None and latest_stat:
+                latest_stat.setdefault("epoch", trainer.current_epoch)
```

Comment:
Is it necessary to add the epoch since we already as default logs it?

Reply:
Yes, you're right. I will remove that.